stm32-modules/include/heater-shaker/heater-shaker/system_task.hpp

```diff
@@ -110,15 +110,15 @@ class SystemTask {
         }
 
         auto cool_message = messages::SetTemperatureMessage{
-            .id = 0, .target_temperature = 0, .from_system = true};
+            .id = 1, .target_temperature = 0, .from_system = true};
```

Comment:
Do these `id` values need to be nonzero before adding to the cache? If not, I think that the intention is a bit clearer if they're left as 0. Doesn't really matter either way tbh.

Reply:
Changed for debugging, but can change back